docs/modules/cli/pages/index.adoc

```diff
@@ -0,0 +1,3 @@
+= Asciidoctor.js CLI Documentation
+
+The Command Line Interface (CLI) for xref:master@asciidoctor.js:ROOT/index.adoc[Asciidoctor.js].
```

Comment:
You don't need to include the version and module in this case as these are the default values. So you can shorten this to:

```
xref:asciidoctor.js::index.adoc[Asciidoctor.js]
```

But since you are fusing the CLI docs with the processor docs, then you just need the module reference (which, for now, you can't drop even though it's ROOT):

```
xref:ROOT:index.adoc[Asciidoctor.js]
```

Reply:
Yep. I haven't figured out how to express it without ending up with a block-like macro (xref::index.adoc[Asciidoctor.js])